.gitignore

```diff
@@ -11,10 +12,12 @@
 /lib
 /local
 /parts
-/var
+/py3
+/share
 /src
 /src/daimler.oira.egg-info
-/py3
+/var
+/var-*
```

Comment:
uh?

Reply:
A personal preference, we can remove that.
I often copy the `var` directory to `var-20221110` or something and this line would also ignore these backups for git.

Reply:
Please try to keep the changes to minimum needed.